deployments/linea-goerli/usdc/migrations/1686918613_configurate_and_ens.ts

```diff
@@ -0,0 +1,226 @@
+      [
+        [configurator.address, cometAdmin.address, rewards.address],
+        [0, 0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)',
+          'setRewardConfig(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata, setRewardConfigCalldata]
+      ]
+    );
+
+    const COMPAmountToBridge = exp(1_000, 18);
+    const USDCAmountToBridge = exp(100_000, 18);
+
+    const goerliActions = [
+      // 1. Set Comet configuration + deployAndUpgradeTo new Comet and set reward config on Linea-Goerli.
+      {
+        contract: lineaMessageService,
+        signature: 'sendMessage(address,uint256,bytes)',
+        args: [bridgeReceiver.address, 0, l2ProposalData]
+      },
+
+      // 2. Approve Goerli's L1StandardBridge to take Timelock's COMP (for bridging)
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [lineaL1TokenBridge.address, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from Goerli to Linea-Goerli Comet using L1StandardBridge
+      {
+        contract: lineaL1TokenBridge,
+        signature: 'bridgeToken(address,uint256,address)',
+        args: [COMP.address, COMPAmountToBridge, rewards.address]
+      },
+      // 4. Approve Goerli's L1StandardBridge to take Timelock's USDC (for bridging)
+      {
+        contract: USDC,
+        signature: 'approve(address,uint256)',
+        args: [lineaL1TokenBridge.address, USDCAmountToBridge]
+      },
+      // 5. Bridge USDC from Goerli to Linea-Goerli Comet using L1StandardBridge
+      {
+        contract: lineaL1TokenBridge,
+        signature: 'bridgeToken(address,uint256,address)',
+        args: [USDC.address, USDCAmountToBridge, comet.address]
+      },
+      // 6. Update the list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(officialMarketsJSON)]
+        )
+      }
+    ];
+
+    const description =
+      '# Configurate Linea-Goerli cUSDCv3 market, set reward config, bridge over USDC and COMP, and update ENS text record.';
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(goerliActions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return true;
+  },
```

Comment:
You'll want to either get rid of this or set it to false. This is automatically set when a migration is enacted.

Enacted migrations are not run before scenarios, meaning you wouldn't be testing this migration.